src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -101,21 +97,23 @@ public void typeEmail(String email) {
     PageObjectLogging.log("typeEmail", "email typed into the field", true);
   }
 
-  public void createAccount() {
-    wait.forElementVisible(createAccountButton);
-    createAccountButton.click();
-    PageObjectLogging.log("createAccount", "Create account button clicked", true);
-    waitForElementNotVisibleByElement(createAccountButton);
+  public void clickRegister() {
+    wait.forElementVisible(registerButton);
+    registerButton.click();
+    PageObjectLogging.log("clickRegister", "Create account button clicked", true);
+    waitForElementNotVisibleByElement(registerButton);
   }
 
   public void createAccountNoEmail(String email, String emailPassword, String userName,
       String password) {
     acceptWikiaAppPolicyNoEmail();
-    MailFunctions.deleteAllEmails(email, emailPassword);
+//    MailFunctions.deleteAllEmails(email, emailPassword);
+waitForValueToBePresentInElementsAttributeByElement(emailField, "value", email);
+
     typeUserName(userName);
     typePassword(password);
-    typeEmail(email);
-    createAccount();
+//    typeEmail(email);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)